### ISSUE TITLE
Problem with making datasets

### ISSUE THREAD

**Issue by LiXm1002** [VisualComputingInstitute/TrackR-CNN#26]
Hi,Track R-CNN is an amazing network. I still have some confusion about the production of data sets in the paper.
"In the beginning, we annotate (as polygons) two segmentation masks per object in the considered dataset.1 The refinement network is first trained on all manually created masks and afterwards fine-tuned individually for each object. "
Why per object need two segmentation masks? What do these two masks label for?

**Comment by pvoigtlaender** [comment]
Hi,
it's not strictly necessary to have 2 segmentation mask per objects. We just did this to get the whole process started. You can also start without annotating mask, and then correct some broken mask, retrain, regenerate masks an iterate.

**Comment by ahnonay** [comment]
As Paul said, the two masks annotated initially are just needed to bootstrap the process. We could also have annotated three per object (meaning more work for the annotators) or only one or zero (meaning less initial information for the network that generates the masks).

Please reopen in case of further questions
